src/aztec3/circuits/kernel/private/c_bind.test.cpp

```diff
@@ -0,0 +1,111 @@
+#include "c_bind.h"
+
+#include <aztec3/circuits/abis/call_context.hpp>
+#include <aztec3/circuits/abis/contract_deployment_data.hpp>
+#include <aztec3/circuits/abis/function_signature.hpp>
+#include <aztec3/circuits/abis/tx_context.hpp>
+#include <aztec3/circuits/abis/tx_request.hpp>
+#include <aztec3/circuits/abis/private_circuit_public_inputs.hpp>
+#include <aztec3/circuits/abis/private_kernel/private_inputs.hpp>
+#include <aztec3/circuits/abis/private_kernel/public_inputs.hpp>
+
+#include <aztec3/circuits/apps/function_execution_context.hpp>
+#include <aztec3/circuits/apps/test_apps/escrow/deposit.hpp>
+
+#include <numeric/random/engine.hpp>
+#include <common/test.hpp>
+#include <gtest/gtest.h>
+
+namespace {
+
+using aztec3::circuits::abis::CallContext;
+using aztec3::circuits::abis::ContractDeploymentData;
+using aztec3::circuits::abis::FunctionSignature;
+using aztec3::circuits::abis::OptionalPrivateCircuitPublicInputs;
+using aztec3::circuits::abis::PrivateCircuitPublicInputs;
+using aztec3::circuits::abis::TxContext;
+using aztec3::circuits::abis::TxRequest;
```

Comment:
Related to my earlier comment - this is an example of what I'd prefer, if possible. Being explicit about what we actually want to use from a namespace, rather than `using` the entire namespace

Reply:
Makes sense!